packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -145,22 +150,13 @@ const runGenerator = (answers, {outputDir, verbose}) => {
         }
     })
 
-    extractTemplate('template-retail-react-app', outputDir)
+    // TODO: Ben (and it should grarefully handle package-not-found error)
```

Comment:
Dumb question: What does fail gracefully mean in this context? the downloadPackage func does not seem to handle when an error happens?

Reply:
Actually, ignore my comment there. Ben's script already exits early if there's an error. That's already good enough for me.

Reply:
Let's remove the code comment if we're not doing it then